scripts/lib/api/TocGrouping.test.ts

```diff
@@ -121,13 +121,38 @@ async function getTocModuleGroups(fp: string): Promise<ModuleGroup[]> {
   return result;
 }
 
-test("Qiskit ToC mirrors index page sections", async () => {
-  validateTopLevelModuleAssumptions();
+async function checkFolder(dirName: string): Promise<void> {
   const indexModuleGroups = await getIndexModuleGroups(
-    "docs/api/qiskit/dev/index.mdx",
+    `docs/api/qiskit${dirName}/index.mdx`,
   );
   const tocModuleGroups = await getTocModuleGroups(
-    "docs/api/qiskit/dev/_toc.json",
+    `docs/api/qiskit${dirName}/_toc.json`,
   );
   expect(indexModuleGroups).toEqual(tocModuleGroups);
+}
+
+describe("Qiskit ToC mirrors index page sections", () => {
+  test("validate assumptions", () => {
+    validateTopLevelModuleAssumptions();
```

Comment:
I could inline the function here, but I like that we get custom error messages by using the current function rather than `expect()`